src/main/frontend/util.cljc

```diff
@@ -1493,6 +1493,20 @@
 
 #?(:cljs
    (defn event-is-composing?
+     "Check if keydown event is a composing (IME) event. 
+      Ignore the IME finishing keycode by default."
+     ([e]
+      (event-is-composing? e true))
+     ([e ignore-finish?]
```

Comment:
what does `ignore-finish?` mean?

Reply:
It's tricky. I updated the PR's comment.
Should change to another name maybe